public/app/percona/dbaas/components/DBCluster/DBCluster.tsx

```diff
@@ -58,6 +61,34 @@ export const DBCluster: FC = () => {
   const [loading, setLoading] = useState(kubernetesLoading);
   const addDisabled = kubernetes?.length === 0 || isKubernetesListUnavailable(kubernetes) || loading;
 
+  const unAvailableK8s = useMemo(
+    () =>
+      kubernetes.filter(
+        (k) => k.status === KubernetesClusterStatus.invalid || k.status === KubernetesClusterStatus.unavailable
+      ),
+    [kubernetes]
+  );
+
+  const availableK8s = useMemo(
+    () =>
+      kubernetes.filter(
+        (k) => k.status !== KubernetesClusterStatus.invalid && k.status !== KubernetesClusterStatus.unavailable
+      ),
+    [kubernetes]
+  );
+
+  useEffect(() => {
+    if (unAvailableK8s.length) {
+      unAvailableK8s.forEach((k) => {
+        appEvents.emit(AppEvents.alertError, [
+          `Unable to load DB clusters for ${k.kubernetesClusterName}. K8s cluster is ${
```

Comment:
Would it possible to add the content to the messages files? I think something like this could work:

```ts
{
    clusterUnavailable: (cluster: string, status: string) => `Unable to load DB clusters for ${cluster}. K8s cluster is ${status}`,
}
```
Then usage would be:
```ts
Messages.clusterUnavailable(k.kubernetesClusterName, Messages.kubernetes.kubernetesStatus[k.status])
```

Reply:
Thanks! Fixed